.gitmodules

```diff
@@ -75,3 +75,10 @@
 [submodule "src/qatzip"]
 	path = src/qatzip
 	url = https://github.com/intel/qatzip.git
+[submodule "src/boost_redis"]
+	path = src/boost_redis
+	url = https://github.com/boostorg/redis.git
```

Comment:
this was added back when you resolved the merge conflict. i opened https://github.com/ceph/ceph/pull/58428 to remove it again